E2_ESRGAN/main.py

```diff
@@ -1,9 +1,32 @@
+import os
 import argparse
+import logging
+from lib import settings, phase_1
+import tensorflow as tf
+
+
+def main(**kwargs):
```

Comment:
Where is this being called? Or is that for later?

Reply:
It will be called from `__name__ == '__main__'` block.